hack/release-docker-images.sh

```diff
@@ -94,3 +99,6 @@ for TAG in "$@"; do
     docker push "${DOCKER_REPO}/api:${TAG}"
   fi
 done
+
+# delete multiarch environment
+trap "docker buildx rm multiarch" EXIT
```

Comment:
That should be declared right after `docker buildx create --use --name multiarch`. Also, trap accept a function name

Reply:
done, thanks :)